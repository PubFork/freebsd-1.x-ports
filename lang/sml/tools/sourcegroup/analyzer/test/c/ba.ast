val it =
  SeqDec
    [MarkDec
       (FctDec
          [MarkFctb
             (Fctb
                {def=FctFct
                       {body=MarkStr
                               (StructStr
                                  (SeqDec [MarkDec (OpenDec [[prim?]],37,43)]),
                                30,47),constraint=NONE,
                        params=[(NONE,
                                 MarkSig
                                   (SigSig
                                      [StrSpec
                                         [(prim?,
                                           MarkSig (VarSig prim?,25,26))]],13,
                                    26))]},name=prim?},10,47)],2,47)]
  : System.Compile.ast
- 