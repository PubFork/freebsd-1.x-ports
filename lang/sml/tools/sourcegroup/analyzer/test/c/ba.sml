functor G (structure D:A) = struct open D end
