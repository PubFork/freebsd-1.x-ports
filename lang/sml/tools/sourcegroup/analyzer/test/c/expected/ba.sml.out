source sml "c/ba.sml"
  import signature A 
  export functor G ;

